usr/sbin/setup-fstab-for-overlayfs

```diff
@@ -1,32 +1,22 @@
-#!/bin/sh
+#!/bin/sh -e
 #
 # Copyright (c) 2019 SUSE Linux GmbH, Nuernberg, Germany
 #
 # Add mountpoint for the /etc overlay and workaround for /var to /etc/fstab
 #
 
 # Already there?
-if [ -e /etc/fstab ] && grep -qE '^overlay[[:space:]]+/etc[[:space:]]' /etc/fstab; then
+if [ -e /etc/fstab ] && grep -qE '^/etc[[:space:]]+/etc[[:space:]]+none[[:space:]]+bind' /etc/fstab; then
 	exit 0 # Do nothing
 fi
 
-# Not a fresh installation?
-if [ -e /etc/fstab.sys ]; then
-	# Let transactional-update handle the migration - doing anything here would be pointless
-	# as it's only written into the overlay anyway and if for some reason transactional-update
-	# is too old, it would result in duplicate /etc entries.
-	exit 0
-fi
+# Migration from old system
+echo "Migrating from /etc overlays to /etc subvolume..."
```

Comment:
It might not just be migration but also initial setup?

Reply:
Thinking about it this script will *only* be called during initial setup. Changed the comment accordingly.